app/src/main/java/org/piramalswasthya/cho/ui/commons/personal_details/PersonalDetailsFragment.kt

```
@@ -465,249 +492,172 @@ class PersonalDetailsFragment : Fragment() {
-                            WorkerUtils.triggerAmritSyncWorker(requireContext())
-                            Toast.makeText(
-                                requireContext(),
-                                getString(R.string.patient_registered_successfully),
-                                Toast.LENGTH_SHORT
-                            ).show()
-
-                            binding.search.text?.clear()
-                            isShowingSearchResults = false
-                            viewModel.filterText("")
+                    saveSuccess = true
+                } catch (e: android.database.sqlite.SQLiteConstraintException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)